MiniTeks/mks-ui
# add deployment files of mks-ui

### What's updated
* added deployment files for deploying on a k8s/OSCP cluster.
* added github action for automatic build of latest image from main branch
* minor fixes in main package

### How to test
0. Make sure redis-db server is up and running. (see mks-server's README.md)
1. Apply the k8s deployment files
```bash
kubectl apply -f k8s
```
2. check if the service is assigned an external IP by OSCP
```bash
k get service -n mks mks-ui --output jsonpath='{.spec.externalIPs[0]}'
```
3. Paste the output of the above step in the chrome.
